.storybook/preview.js

```diff
@@ -8,23 +8,15 @@ import '@cds/core/icon/register.js';
 
 import { loadCoreIconSet, loadEssentialIconSet } from '@cds/core/icon';
 import { setCompodocJson } from '@storybook/addon-docs/angular';
+import { applicationConfig } from '@storybook/angular';
+import { provideAnimations } from '@angular/platform-browser/animations';
 
-import previewStyles from 'raw-loader!./public/preview.css';
 import docs from '../documentation.json';
-import resetStyles from 'raw-loader!../node_modules/@cds/core/styles/module.reset.min.css';
-import coreTokens from 'raw-loader!../node_modules/@cds/core/global.min.css';
-import darkThemeCoreStyles from 'raw-loader!../node_modules/@cds/core/styles/theme.dark.min.css';
-
-// Styles that should be watched/reloaded
-import clrUiStyles from 'raw-loader!sass-loader!../projects/ui/src/clr-ui.scss';
-
 import { THEMES } from './helpers/constants';
 
 const privateModifier = 121;
 const cdsThemeAttribute = 'cds-theme';
-const styleElement = addStyleElement();
-
-const cdsCoreStyles = [previewStyles, resetStyles, coreTokens, darkThemeCoreStyles, clrUiStyles];
+addStyleElement();
```

Comment:
This should be removed because it's not needed anymore. (Please remember to remove the function declaration as well.)